src/styles/card.scss

```diff
@@ -104,20 +104,56 @@ $fd-card-header-outline-offset: 0.0625rem !default;
     }
 
     &:last-child {
+      padding: var(--fdCard_Bottom_Positioned_Header_Padding);
       border-bottom: none;
       border-top: $fd-card-header-border;
-      border-radius: 0 0 $fd-card-header-border-radius $fd-card-header-border-radius;
+      border-radius: 0 0 $fd-card-border-radius $fd-card-border-radius;
 
       @include fd-focus() {
         &::before {
           top: $fd-card-header-outline-offset * 2;
           bottom: $fd-card-header-outline-offset;
-          border-radius: 0 0 $fd-card-header-border-radius $fd-card-header-border-radius;
+          border-radius: 0 0 $fd-card-border-radius $fd-card-border-radius;
         }
       }
     }
   }
 
+  &__footer {
+    @include fd-reset();
+    @include fd-flex-vertical-center();
+
+    padding: var(--fdCard_Footer_Padding);
+    background: $fd-card-default-body-background-color;
+    border-bottom: $fd-card-header-border;
+    border-radius: 0 0 $fd-card-border-radius $fd-card-border-radius;
+
+    &-actions {
+      @include fd-reset();
+
+      @include fd-flex() {
+        justify-content: flex-end;
+      }
+
+      width: 100%;
+
+      &-item {
+        @include fd-reset();
```

Comment:
in this case you don't need a reset as you are using the class like this:
```
<button class="fd-button fd-button--positive fd-card__footer-actions-item">
```

if you add a reset for `fd-card__footer-actions-item` it might reset some of the rules set by `fd-button` and `fd-button--positive`

Reply:
Actually that should not have been that way. Button should be separate instance from the elements that are wrapping it